Cargo.toml

```diff
@@ -58,6 +58,7 @@ tempfile = "3.2.0"
 [features]
 deltalake = ["dep:deltalake"]
 flightsql = ["dep:arrow-flight", "dep:tonic"]
+experimental-flightsql-server = ["dep:arrow-flight", "dep:tonic"]
```

Comment:
Given this is early stage, you need to explicitly opt into the experimental feature